vendir.s2.yml

```diff
@@ -0,0 +1,15 @@
+apiVersion: vendir.k14s.io/v1alpha1
+kind: Config
+directories:
+  - path: helm/cilium
+    contents:
+      - path: .
+        directory:
+          path: vendor/cilium/install/kubernetes/cilium
+        ignorePaths:
```

Comment:
We even could remove this `ignorePaths` directive from this stage 2 file and remove unwanted files in the `customization.sh` file so everything that changes the modified upstream chart is one place.

Reply:
We talked about it and it isn't that easy to write a script that would remove everything while ignoring paths like `templates/**/networkpolicy.yaml`